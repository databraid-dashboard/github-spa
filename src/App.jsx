/* eslint-disable import/no-named-as-default */
import React from 'react';
import { Container } from 'semantic-ui-react';
import RepoList from './components/RepoList/RepoList';
import Login from './components/Login/Login';
import Organizations from './components/Organizations/Organizations';
import Dashboard from './components/Dashboard/Dashboard';
import './App.css';

const App = () => (
  <Container>
    <Login />
    <Organizations />
    <RepoList />
    <Dashboard />
  </Container>
);

export default App;
