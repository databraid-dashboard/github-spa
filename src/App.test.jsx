import React from 'react';
import toJson from 'enzyme-to-json';
import { shallow } from 'enzyme';
import App from './App.jsx';

describe('App component', () => {
  it('should render a component with props as specified ', () => {
    const component = shallow(
      <App />,
    );
    expect(toJson(component)).toMatchSnapshot();
  });
});
