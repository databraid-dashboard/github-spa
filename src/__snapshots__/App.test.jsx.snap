// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App component should render a component with props as specified  1`] = `
<Container>
  <Connect(Component) />
  <Connect(Organizations) />
  <Connect(RepoList) />
  <Connect(Component) />
</Container>
`;
